app/src/main/res/values/faqs.xml

```diff
@@ -9,13 +9,13 @@
-  <string name="faq_answer_2"><![CDATA[<p>Once a profile is deleted:</p> <p><br></p> <p> <li> The profile cannot be recovered. </li> </p> <p> <li> Profile information such as name, photos, and progress will be permanently deleted. </li> </p> <p><br></p> <p>To delete a profile (excluding the <u>Administrator\'s</u>):</p> <p>1. From the Administrator\'s Home Page, tap on the menu button on the top left.</p> <p>2. Tap on <strong>Administrator Controls</strong>.</p> <p>3. Tap on <strong>Edit Profiles</strong>.</p> <p>4. Tap on the Profile you would like to delete.</p> <p>5. At the bottom of the screen, tap <strong>Profile Deletion</strong>.</p> <p>6. Tap <strong>Delete</strong> to confirm deletion.</p><p><br></p><p>Note: Only the <u>Administrator</u> is able to manage profiles.</p>]]></string>
+  <string name="faq_answer_1"><![CDATA[<p>If it is your first time creating a profile and not have a PIN: </p> <p> 1. From the Profile Chooser, tap on <strong>Set up Multiple Profiles</strong>. </p> <p> 2. Create a PIN and <strong>Save</strong>. </p> <p> 3. Fill in all fields for the profile. </p> <ul><li dir="rtl> (Optional) Upload a photo. </li><li dir="rtl> Enter a name. </li><li dir="rtl> (Optional) Assign a 3-digit PIN. </li> </ul> <p> 4. Tap <strong>Create</strong>. This profile is added to your Profile Chooser! <br/> <br/> If you have created a profile before and have a PIN: </p> <p> 1. From the Profile Chooser, tap on <strong>Add Profile</strong>. </p> <p> 2. Enter your PIN and tap <strong>Submit</strong>. </p> <p> 3. Fill in all fields for the profile. </p> <ul><li dir="rtl> (Optional) Upload a photo. </li><li dir="rtl> Enter a name. </li><li dir="rtl> (Optional) Assign a 3-digit PIN. </li> </ul> <p> 4. Tap <strong>Create</strong>. This profile is added to your Profile Chooser! <br> <br> Note: Only the <u>Administrator</u> is able to manage profiles.</p>]]></string>
+  <string name="faq_answer_2"><![CDATA[<p>Once a profile is deleted:</p> <p><br></p> <p><li dir="rtl> The profile cannot be recovered. </li> </p> <p><li dir="rtl> Profile information such as name, photos, and progress will be permanently deleted. </li> </p> <p><br></p> <p>To delete a profile (excluding the <u>Administrator\'s</u>):</p> <p>1. From the Administrator\'s Home Page, tap on the menu button on the top left.</p> <p>2. Tap on <strong>Administrator Controls</strong>.</p> <p>3. Tap on <strong>Edit Profiles</strong>.</p> <p>4. Tap on the Profile you would like to delete.</p> <p>5. At the bottom of the screen, tap <strong>Profile Deletion</strong>.</p> <p>6. Tap <strong>Delete</strong> to confirm deletion.</p><p><br></p><p>Note: Only the <u>Administrator</u> is able to manage profiles.</p>]]></string>
```

Comment:
@BenHenning Can you confirm if this approach is correct or not?

Reply:
@BenHenning can you please confirm on this as it will unblock me.

Reply:
Followed up in a few comments below.